packages/keyring-eth-hd/package.json

```diff
@@ -57,11 +57,11 @@
     "@lavamoat/preinstall-always-fail": "^2.1.0",
     "@metamask/auto-changelog": "^3.4.4",
     "@metamask/bip39": "^4.0.0",
-    "@metamask/eth-hd-keyring": "4.0.1",
     "@ts-bridge/cli": "^0.6.1",
     "@types/jest": "^29.5.12",
     "deepmerge": "^4.2.2",
-    "jest": "^29.5.0"
+    "jest": "^29.5.0",
+    "old-hd-keyring": "npm:@metamask/eth-hd-keyring@^4.0.1"
```

Comment:
This package here was overriding the current version of `@metamask/eth-hd-keyring` confusing TS. With this line we can import the package with a different name

Reply:
The version `4.0.1` was pinned, but `yarn lint` was failing and I guess that `^4.0.1` serves the same purpose